app/views/errors/internal_server_error.html.erb

```diff
@@ -18,4 +18,18 @@
       </p>
     </div>
   </div>
+
+  <% if Rails.env.development? %>
```

Comment:
Just to note that you may have some trouble getting past RSpec + Undercover with your changes to the controller action.

You might have more success leaving the controller untouched and doing this in the view:

```suggestion
  <% if Rails.env.development? && exception = request.env["action_dispatch.exception"]&.cause %>
```